### PR TITLE
Feature/tweet box

### PR DESCRIPTION
**Pull request by shivkumarganesh**
**Hi! I have completed the following:-**
1. Added Text Box for creating a tweet
2. Moved the dashboard to App-Container (Which is an app shell)
3. Done partial refactoring.

**In Progress**
1. Refactoring is in progress
2. Testcases are being written
3. Style Fixes needed (Externalization to stylesheet)

### CODE REVIEW

File: app/src/config/api-config.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/**
+ * @author Shiv Kumar Ganesh <shivkumar.ganesh@sjsu.edu>
+ */
+export const API_URL = {
+    CREATE_TWEET : '/api/create',
+    DELETE_TWEET: '',
+    GET_TWEETS: '',
```
Comment (shannonphu) [inline app/src/config/api-config.ts:7]: Do we need a separate endpoint for get vs retrieve vs list tweets? Does this mean a particular tweet might be like clickable to retrieve one?

File: server/config/twitter-config.js [hunk 2]
```diff
@@ -4,10 +4,10 @@
 var Twit = require('twit')
  
 var Twitter = new Twit({
-  consumer_key:         '',
-  consumer_secret:      '',
-  access_token:         '',
-  access_token_secret:  '',
+  consumer_key:         'YSqTdwKJ5T9rw061P1yZPvew8',
```
Comment (shannonphu) [inline server/config/twitter-config.js]: This config was commited.

**Review by shannonphu — CHANGES_REQUESTED** [review]

File: server/config/twitter-config.js (hunk 2 above)
Comment (shivkumarganesh) [reply server/config/twitter-config.js]: I am removing it.

File: app/src/config/api-config.ts (hunk 1 above)
Comment (shivkumarganesh) [reply app/src/config/api-config.ts:7]: Let it be there. I was thinking that if someone clicked on tweet do we pull information and edit or update an existing tweet.

**Review by shannonphu — APPROVED** [review]

File: app/src/config/api-config.ts (hunk 1 above)
Comment (jocelynbaduria) [reply app/src/config/api-config.ts:7]: I think get is the same as the features of retrieve. 
This is our requirement from this HW "Develop a simple twitter service that implement any Twitter API to to programmatically create, retrieve and delete a Tweet in the Twitter Sandbox environment."

**Review by jocelynbaduria — COMMENTED** [review]
Looks good

**Pull request merged**